### PR TITLE
[MM-39311] Pass session token to call made by the buitin app

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
- beed3582ea2df9430be2f8169fa8bc39377f27c8: Some call trigger by the builtin app need a userToken to pass it to the app. The call not expand it and pass it along.
- 4c221baf953dfca240072e7208e07ce5c0024116: Some formatting fixes
- 3d3b6101d6fa57da5a1d35438ce28542e998ff18: Don't longer show an error in the logs when kubless upstream is not configured

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-39311

### CODE REVIEW

**Comment by DHaussermann** [comment]
@hanzei this is not installable for me. 

I see the normal manifest when I start the test-app from `master` but, when I switch to this branch and start the test-app the manifest appears invalid and the app can't be installed. 
image

Is there an extra deployment step needed here?

**Comment by DHaussermann** [comment]
Sorry @hanzei disregard previous comment. I didn't realize I have to append the file `.json` file extension now. 
I see the app install working. Testing is ongoing.

**Review by larkox — APPROVED** [review]

File: server/builtin/debug_clean.go [hunk 1]
```diff
@@ -28,8 +28,7 @@ func (a *builtinApp) debugClean() handler {
 				Call: &apps.Call{
 					Path: pDebugClean,
 					Expand: &apps.Expand{
-						AdminAccessToken: apps.ExpandAll, 
-						Locale:           apps.ExpandAll,
+						ActingUser: apps.ExpandSummary,
```
Comment (hanzei) [inline server/builtin/debug_clean.go:31]: The access token is not used

File: upstream/upkubeless/upstream.go [hunk 2]
```diff
@@ -31,7 +32,7 @@ var _ upstream.Upstream = (*Upstream)(nil)
 
 func MakeUpstream() (*Upstream, error) {
 	_, err := kubelessutil.BuildOutOfClusterConfig()
-	if os.IsNotExist(err) {
+	if os.IsNotExist(err) || kubelessclient.IsEmptyConfig(err) {
```
Comment (levb) [inline upstream/upkubeless/upstream.go:35]: 👍

**Review by levb — APPROVED** [review]

**Comment by levb** [comment]
@DHaussermann I am going to go ahead and merge this, as more work depends on it, to avoid multi-merge situation

File: server/proxy/expand.go [hunk 3]
```diff
@@ -45,28 +45,17 @@ func (p *Proxy) expandContext(in Incoming, app apps.App, base *apps.Context, exp
...
 	}
 
-	var userSession *model.Session
 	if expand.AdminAccessToken != "" {
 		if !app.GrantedPermissions.Contains(apps.PermissionActAsAdmin) {
 			return emptyCC, utils.NewForbiddenError("%s does not have permission to %s", app.AppID, apps.PermissionActAsAdmin)
 		}
-		err := utils.EnsureSysAdmin(p.conf.MattermostAPI(), cc.ActingUserID)
+
+		err = in.ensureAdminToken(mm)
```
[the remark this answers is not in the document]
Comment (hanzei) [reply server/proxy/expand.go:53]: Good point. The rework of `Incoming` will remove the admin checks.

**Pull request merged**